.github/workflows/test-bash-functions.yml

```diff
@@ -8,11 +8,11 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Run release functions test
-        run: "bash scripts/test_release_functions.sh"
+        run: "bash scripts/tests/test_release_functions.sh"
```

Comment:
I do not know if that is a problem. But locally I can not execute those shell scripts located in tests. 

The same is true for the referenced files but this may be fine as you execute them from within other scripts? IDK.

Reply:
Weird, I can execute them directly... 
What was the error?

Reply:
No error at all. The files are listet with ls -a as not executable. Maybe just a local problem.. Weird.